[Docs] Sphinx book theme change (#3522)
The-OpenROAD-Project/OpenROAD: docs/contrib/DeveloperGuide.md

database files. For consistency, these functions are all provided by the OpenROAD
framework.

> Tools should package all states in a single class. An instance of each

Reviewer: ```suggestion
Tools should package all of their state in a single class. An instance of each
```

tool class, an initialization function that is called after all of
the tools have been made, and a function to delete the tool. This small
header does **not** include the class definition for the tool so that

> OAOAOAthe OpenROAD framework does not have to know anything about the tool

Reviewer 2: ```suggestion
the OpenROAD framework does not have to know anything about the tool
```

examples. Use swig to define internal functions to C++ functionality.

Tcl files can be included by encoding them in CMake into a string that

> is evaluated at run time (See `Resizer::init()` [here](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/rsz/src/Resizer.cc)).

Reviewer: ```suggestion
is evaluated at run time (See [`Resizer::init()`](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/rsz/src/Resizer.cc)).
```

Tool commands should be documented in the top-level OpenROAD `README.md`
file. Detailed documentation should be the `tool/README.md` file.

> ## Tool Flow Namespace

Reviewer 3: @vvbandeira Should the gui be added here?
